coalib/settings/Section.py

```diff
@@ -253,3 +253,7 @@ def delete_setting(self, key):
         :param key: The key of the setting to be deleted
         """
         del self.contents[key]
+
+    @property
+    def name_str(self):
```

Comment:
i thik you need to document properties too. as parameters are missing here, you should explain what it returns, as in
:return: